imagetest/test_suites/image_boot/shutdown_test.go

```diff
@@ -0,0 +1,31 @@
+package imageboot
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+const shutdownTime = 110 // about 2 minutes
+
+// TestGuestShutdownScript test that shutdown scripts can run for around two minutes
+func TestGuestShutdownScript(t *testing.T) {
+	_, err := os.Stat(markerFile)
+	if os.IsNotExist(err) {
+		// first boot
+		if _, err := os.Create(markerFile); err != nil {
+			t.Fatalf("failed creating marker file: %v", err)
+		}
+		t.Fatal("marker file does not exist")
+	}
```

Comment:
2 issues

1. this test AND `TestGuestReboot` are both going to try to create the same marker file on the same VM. this could be race condition where that test creates the file, so this test thinks it is already the second boot.
2. we already have an implicit marker file on this test - the file `/shutdown.txt`. let's just use that since we know it only gets created when the shutdown script runs. and since we already have error handling for it not existing, we can write a simpler test by just omitting this section entirely.

```suggestion

```